lib/LLVMIRCodeGen/GlowJIT.cpp

```diff
@@ -135,7 +116,7 @@ GlowJIT::GlowJIT(llvm::TargetMachine &TM)
                      return RTDyldObjectLinkingLayer::Resources{
                          std::make_shared<SectionMemoryManager>(), resolver_};
                    }),
-#elif LLVM_VERSION_MAJOR > 6
+#else // LLVM_VERSION_MAJOR > 6
```

Comment:
This comment seems incorrect

Reply:
Hmm. The comment refers to versions for which the following section should work.
Do you mean that I'm not actually enforcing the condition anywhere?

Reply:
yes the condition is not enforced anywhere.

Reply:
I was referring to the other side of this branch not having anything to do with `LLVM_VERSION_MAJOR > 6`, i.e. it's `#if FACEBOOK_INTERNAL && LLVM_VERSION_PATCH < 20181009` vs. this `#else`